src/include/bitstring_inline.h

```diff
@@ -94,6 +94,16 @@ __bit_set(uint8_t *bitf, uint64_t bit)
 	bitf[__bit_byte(bit)] |= __bit_mask(bit);
 }
 
+/*
+ * __bit_set_mask --
+ *	Apply a mask to a range of bits at a given index.
+ */
+static WT_INLINE void
+__bit_set_mask(uint8_t *bitf, uint64_t index, uint8_t mask)
```

Comment:
Where is this used? I see you using `bit_nset`. My search of this diff is not showing anything (and I've expanded all of github's "large diffs") If it isn't used anymore you should remove it.

Reply:
Remove in: ba9df39c3